kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/WEB-INF/web.xml

```diff
@@ -17,9 +17,10 @@
     <web-resource-collection>
       <web-resource-name>Swagger web resources</web-resource-name>
       <url-pattern>/services/rest/server/swagger.json</url-pattern>
+      <url-pattern>/services/rest/server/files/*</url-pattern>
       <http-method>GET</http-method>
     </web-resource-collection>
-    <!-- No authentication. Swagger resources should be accessible without authentication. -->
+    <!-- No authentication. Swagger resources should be accessible without authentication and files (form rendering). -->
```

Comment:
Maybe should be `Swagger resources and files (form rendering) should be accessible without authentication.`